products/centos-stream.md

```diff
@@ -0,0 +1,41 @@
+---
+title: CentOS Stream
+category: os
+iconSlug: centos-stream
+permalink: /centos-stream
+versionCommand: lsb_release --release
+releasePolicyLink: https://wiki.centos.org/About/Product
+releaseLabel: "CentOS Stream __RELEASE_CYCLE__"
+activeSupportColumn: true
+releaseDateColumn: true
+
+identifiers:
+-   cpe: cpe:/o:centos:centos
+-   cpe: cpe:2.3:o:centos:centos
+
+releases:
+-   releaseCycle: "9"
+    releaseDate: 2021-09-15
+    support: 2027-05-31
+    eol: 2027-05-31
+    latest: "9"
+    link: https://blog.centos.org/2021/12/introducing-centos-stream-9/
+
+-   releaseCycle: "8"
+    releaseLabel: "CentOS Stream 8"
```

Comment:
This is not necessary anymore (it was because both CentOS 8 and CentOS Stream 8 was documented on the same page).